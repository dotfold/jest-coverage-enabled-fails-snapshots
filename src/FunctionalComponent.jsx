
import React from 'react'

const FunctionalComponent = (props) =>
  <div>
    {props.name}
    This is a functional component
  </div>

export default FunctionalComponent
