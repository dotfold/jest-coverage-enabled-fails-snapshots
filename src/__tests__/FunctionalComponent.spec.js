
import React from 'react'
import renderer from 'react-test-renderer'

import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import FunctionalComponent from '../FunctionalComponent'

describe('<FunctionalComponent />', () => {
  it('should render correctly with react-test-renderer', () => {
    const wrapper = renderer.create(
      <FunctionalComponent />
    ).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
  it('should render correctly with enzyme', () => {
    const wrapper = shallow(
      <FunctionalComponent />
    )
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
