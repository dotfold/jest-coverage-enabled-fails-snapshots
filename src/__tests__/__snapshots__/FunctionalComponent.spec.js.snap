exports[`<FunctionalComponent /> should render correctly with enzyme 1`] = `
<div>
  This is a functional component
</div>
`;

exports[`<FunctionalComponent /> should render correctly with react-test-renderer 1`] = `
<div>
  This is a functional component
</div>
`;
